src/mocks/streams.ts

```diff
@@ -0,0 +1,79 @@
+import { StreamResponse } from "@types";
+
+const now = new Date();
+const oneHourAgo = new Date(now.getTime() - 60 * 60 * 1000);
+const twoHoursLater = new Date(now.getTime() + 2 * 60 * 60 * 1000);
+const fourHoursLater = new Date(now.getTime() + 4 * 60 * 60 * 1000);
+const sixHoursLater = new Date(now.getTime() + 6 * 60 * 60 * 1000);
+const tomorrow = new Date(now.getTime() + 24 * 60 * 60 * 1000);
+const ttl = new Date(now.getTime() + 30 * 24 * 60 * 60 * 1000); // 30 days from now
+
+export const mockStreams: StreamResponse[] = [
+  {
+    id: "stream1",
+    streamerId: "streamer1",
+    channelId: "UC1234567890",
+    platform: "youtube",
+    title: "【LIVE】Morning Gaming Stream - Apex Legends Ranked",
+    thumbnail: "https://via.placeholder.com/640x360/FF6B6B/FFFFFF?text=Stream+1",
```

Comment:
Use the real thumbnail of this video https://www.youtube.com/live/YkySMsypko8?si=x7acbWNf5H_9DIXf

Also update for other streams